browse/templates/base.html

```diff
@@ -34,12 +34,13 @@
         <div class="column" id="cu-logo">
           <a href="https://www.cornell.edu/"><img src="{{ url_for('static', filename='images/icons/cu/cornell-reduced-white-SMALL.svg') }}" alt="Cornell University" /></a>
         </div>
-        {%- set rd_int = request_datetime.strftime("%Y%m%d%H%M")|int -%}
-        {%- if rd_int >= 202305010100 and rd_int <= 202307170100 -%}
+        {%- if config["BROWSE_MINIMAL_BANNER_ENABLED"] and rd_int >= 202310010100 and rd_int <= 202312300100 -%}
         <div class="column banner-minimal">
-            <a href="https://info.arxiv.org/hiring/">We are hiring</a></p>
+            <a href="https://info.arxiv.org/about/index.html">To the cloud!</a></p>
```

Comment:
@alisonhofer might have a different take, but arXiv's tone is usually more matter of fact. What about "Moving to the cloud" or "Server stability update"?